test/mcs/test_defmcs_sobol.jl

```diff
@@ -68,6 +68,7 @@ results_disk[!,2] = Symbol.(results_disk[!,2])
 
 # do some analysis
 E = CSVFiles.load(joinpath(output_dir, "emissions_E.csv")) |> DataFrame
+results = analyze(si, E[1:60:end, 3]; progress_meter = false, N_override = 100)
 results = analyze(si, E[1:60:end, 3]; num_resamples = 10_000, conf_level = 0.95)
```

Comment:
Do we want to use `progress_meter=false` everywhere in the tests that use `analyze`? I think David said he prefers that

Reply:
also why are there two calls here? is it for test coverage? is it not possible to hit all the lines with one call because of the `N_override` if statements or something?

Reply:
Yes you're right we should make it false in all those cases.  I'm not sure off the top of my head maybe I use the `results` from line 72 in a later test?

Reply:
I guess I don't, so think I just wanted to check out both branches of setting N.

Reply:
cool makes sense!